app/assets/javascripts/config.js.coffee.erb

```diff
@@ -3,6 +3,17 @@
 
   csrfToken = document.querySelector?('meta[name=csrf-token]')?.content
 
+  query = window.location.search
+  match = /echo_env=(\w+)/.exec(query)
+  echo_env = if match then match[1] else '<%= Rails.configuration.echo_env %>'
+
+  if echo_env == 'partnertest'
+    browse_scaler_url = "https://api-test.echo.nasa.gov/browse-scaler/browse_images"
```

Comment:
Can you still use Rails.configuration for this?  Something like:

    browse_scaler_url = "<%= Rails.configuration.services['earthdata']['partnertest']['echo_root'] %>/browse-scaler/browse_images"

That way we still maintain the ECHO roots in a single place rather than hard-coding some places and using rails config elsewhere.